HCIDOCS-139: Fix nohup command in network config for IPI/bare metal installation (#82678)
openshift/openshift-docs: modules/ipi-install-configuring-networking.adoc

.Procedure

> . Export the bare-metal network NIC name by running the following command:

Author: I think this is wrong. "network" is a noun.

> . Export the bare-metal network NIC name by running the following command:

Reviewer: I agree in this case.

+
[NOTE]
====
The SSH connection might disconnect after executing these steps.

> The IPv6 address can be any address provided it is not routable through the bare-metal network.

Author: I think this is wrong. "network" is a noun.

> The IPv6 address can be any address provided it is not routable through the bare-metal network.

Reviewer: > any address provided

I read this as _any address that you provided_... Maybe:
```suggestion
The IPv6 address can be any address that is not routable through the bare-metal network.
```

Ensure that UEFI is enabled and UEFI PXE settings are set to the IPv6 protocol when using IPv6 addressing.
====

. Optional: If you are deploying with a provisioning network, configure the IPv4 address on the provisioning network connection by running the following command:
+
[source,terminal]
----
$ nmcli connection modify provisioning ipv4.addresses 172.22.0.254/24 ipv4.method manual
----

. SSH back into the `provisioner` node (if required) by running the following command:
+
[source,terminal]
----
# ssh kni@provisioner.<cluster-name>.<domain>
----

> . Verify the connection bridges have been properly created by running the following command:

Reviewer: ```suggestion
. Verify that the connection bridges have been properly created by running the following command:
```
